### PR TITLE
Added exclude-categories flag

### LINKED ISSUE #2161: Exclude category of queries
Good afternoon.
**Is your feature request related to a problem? Please describe.**
I would like to be able to perform exclude certain categories of requests. For example I believe I don't want to check Privileged containers in Kubernetes manifests.

**Describe the solution you'd like**
It should be some flag that would allow a list of categories, separated by commas and then queries from that category would not run on my source code.

**Describe alternatives you've considered**
Alternatively it could be a command to display all available check categories and the option to specify only specified categories for checking.

### PR DESCRIPTION
**Pull request by felipe-avelar**
Closes #2161 

**Proposed Changes**

- Now it is possible to exclude queries by its category
- Refactor to solve issues from sonar
- Refactor to reorganize code elements
I submit this contribution under Apache-2.0 license.

### CODE REVIEW

**Comment by rogeriopeixotocx** [comment]
@felipe-avelar please resolve conflicts

File: internal/console/scan.go [hunk 1]
```diff
@@ -205,6 +210,15 @@ func initScanCmd() {
 			"can be provided multiple times or as a comma separated string\n"+
 			"example: 'fec62a97d569662093dbb9739360942f...,31263s5696620s93dbb973d9360942fc2a...'",
 	)
+	scanCmd.Flags().StringSliceVarP(
+		&excludeCategories,
+		"exclude-categories",
+		"",
+		[]string{},
+		"exclude categories by providing its name\n"+
```
Comment (rogeriopeixotocx) [inline internal/console/scan.go:218]: ... `providing it's name\n`

**Review by rogeriopeixotocx — CHANGES_REQUESTED** [review]

**Review by rogeriopeixotocx — APPROVED** [review]
LGTM

**Review by joaoReigota1 — APPROVED** [review]
LGTM

**Review by ruigomescx — APPROVED** [review]

**Pull request merged**